docs/usermanual.md

```diff
@@ -144,6 +117,25 @@ E.g.:
 }
 ```
 
+#### Commands
```

Comment:
It seems you have changed the order of sections... why?

Reply:
The order of section has been changed for incorporation of item no #3 of issue #216